src/tss2-tcti/tcti-tbs.c

```diff
@@ -325,4 +327,4 @@ const TSS2_TCTI_INFO*
 Tss2_Tcti_Info (void)
 {
     return &tss2_tcti_info;
-}
\ No newline at end of file
+}
```

Comment:
Nit: superfluous white space change, split to another patch.

Reply:
There is no change. This is just the git way of noting that the file does not ends with '\n'
It looks different in gitub GUI. The git diff in terminal looks like this:
```
@@ -325,4 +327,4 @@ const TSS2_TCTI_INFO*
 Tss2_Tcti_Info (void)
 {
     return &tss2_tcti_info;
-}
\ No newline at end of file
+}
```
See https://mrkiffie.com/2016/newlines-and-version-control/

Reply:
Freaking UI's